### ISSUE TITLE
dependency error (0.18.0)

### ISSUE THREAD

**Issue by matsu-hide8** [phpactor/phpactor#1341]
The following error occurred when I tried to install phpactor 0.18.0.

```
  Problem 1
    - Root composer.json requires phpactor/phpactor ^0.18.0 -> satisfiable by phpactor/phpactor[0.18.0].
    - phpactor/phpactor 0.18.0 requires phpactor/completion-extension ^0.2.5 -> found phpactor/completion-extension[dev-master, 0.2.x-dev (alias of dev-master)] but it does not match your minimum-stability.
```

It seems to depend on phpactor/completion-extension 0.2.5, which has not been released yet.

**Comment by dantleech** [comment]
that's odd. is this installing as a standalone?

errornously `composer.json` in 0.18 is currently set to `minimum-stability: dev, prefer-stable: true` which means it will use a tagged version if it exists, otherwise use the branch alias.

that said, `composer install` should work.

**Comment by matsu-hide8** [comment]
Yes. I installed it as standalone.

I was able to install it by adding `minimum-stability: dev` to my composer.json.

I understand that Phpactor is in the development stage.
Do I need this setting in my composer.json as well?

**Comment by dantleech** [comment]
No, it's a problem with the release it should have minstability "stable"

**Comment by matsu-hide8** [comment]
Finaly it will be stable, but for now the user needs to set `"minimum-stability": "dev"`?
Or something bug??

**Comment by dantleech** [comment]
I will need to tag a new release. Normally this isn't an issue as it's expected that the package is installed "standalone"

**Comment by dantleech** [comment]
but I think you should be able to install with stable deps now, so in that case you can close this ticket.

**Comment by matsu-hide8** [comment]
Has 0.18.1 been updated yet?

**Comment by dantleech** [comment]
Just tagged it.

**Comment by matsu-hide8** [comment]
I was able to install without `minimum-stability: dev`.
Thanks!
